protocols/v2/roles-logic-sv2/src/handlers/common.rs

```diff
@@ -38,14 +36,14 @@ use common_messages_sv2::{
 use const_sv2::*;
 use core::convert::TryInto;
 use std::sync::Arc;
-use tracing::{debug, error, info, trace};
+use tracing::{debug, error, info};
 
 /// see [`SendTo_`]
 pub type SendTo = SendTo_<CommonMessages<'static>, ()>;
 
 /// A trait that is implemented by the downstream. It should be used to parse the common messages
 /// that are sent from the upstream to the downstream.
-pub trait ParseUpstreamCommonMessages<Router: CommonRouter>
+pub trait ParseUpstreamCommonMessages
```

Comment:
```suggestion
pub trait ParseCommonMessagesFromUpstream
```

we should take the opportunity of this PR to clear up confusion on naming

Reply:
oops sorry this is already being done in a future commit, sorry for the noise